### DISCUSSION TITLE
RSS feed page

### DISCUSSION THREAD

**Discussion by rivetgeekwil** [CaiJimmy/hugo-theme-stack#607] [Ideas]
It would be nice to have an RSS feed page similar to this: https://pakstech.com/blog/hugo-rss/#list-available-rss-feeds
I tried cobbling it together, but I'm not proficient enough with Hugo or CSS to get it working correctly.

EDIT: I was able to get it working as a static page. You can see the example here: https://aggregatecognizance.com/rss-feeds/

**Comment by CaiJimmy** [comment]
Sorry, no plan to implement that 😥

**Reply by Darthagnon** [reply]
Quick suggestion RE: RSS feeds, it's easy to make the general RSS page look good. See [this commit](https://github.com/AmazingRise/hugo-theme-diary/commit/31b47c466a683c1359ae4061b7dbf2e6a68bac74), where I basically copypasted [this styling from Typlog](https://lepture.com/en/2019/rss-style-with-xsl). It's 1 or 2 files, and makes RSS feed human-readable when clicked on. I'm not sure what Stack's RSS feed looks like, but I haven't seen many sites use XSL styling.
